### ISSUE TITLE
BUNQ integration

### ISSUE THREAD

**Issue by danrminds** [teamforus/forus#103]
- [ ] Top up a fund
- [ ] Connection to the BUNQ API
- [ ] Trigger transaction to shopkeeper
- [ ] Monitor transaction costs

**Comment by maxvisser** [comment]
the API from another big dutch bank is also in a prototype stage (not production ready). if there is time we could look at it and try to write a connection it.

After that we could start a conversation with ING about next steps (starting a production connection).

**Comment by maxvisser** [comment]
I think topping up a fond and ING research is for after november. Maybe split this issue up?

**Comment by danrminds** [comment]
Yes, we can create separate tasks/issues for researching ING and the other bank. BUNQ should remain here with the deadline of 1 nov 2018

**Comment by danrminds** [comment]
- [ ] New keys for BUNQ
